monai/networks/utils.py

```diff
@@ -560,6 +563,7 @@ def convert_to_torchscript(
     device: torch.device | None = None,
     rtol: float = 1e-4,
     atol: float = 0.0,
+    use_trace: bool = False,
```

Comment:
I think you didn't add support for this new arg in the `ckpt_export` API? May also need to add `inputs`?

Thanks.

Reply:
I have added the `ir` and `input_shape` parameters to the `ckpt_export` API. Could you please take a look at it and tell me if it looks fine with you?

Thanks.